videocall-client/src/media_devices/media_device_list.rs

```diff
@@ -144,8 +144,8 @@ impl MediaDeviceList {
         let on_loaded = self.on_loaded.clone();
         let on_audio_selected = self.audio_inputs.on_selected.clone();
         let on_video_selected = self.video_inputs.on_selected.clone();
-        let audio_input_devices = Arc::clone(&self.audio_inputs.devices);
-        let video_input_devices = Arc::clone(&self.video_inputs.devices);
+        let audio_input_devices = Rc::clone(&self.audio_inputs.devices);
```

Comment:
@griffobeid did clippy do this?

Reply:
Yeah https://rust-lang.github.io/rust-clippy/master/index.html#/Arc

Reply:
neat!